nss-day-cohort-47/holiday-road-tundra-travelers
# cc-sidenav

-To test it follow these instructions
Click the logo on the top left of the screen, it should open up the navbar on that side with a short animation.
-What's been changed
Instead of having the nav be in a row almost in the center of the screen I have created a new newbar that opens from the side of the page. I also changed the name's for destinations and bizarries in the javascript for a moment, it makes our lives easier while we are writing the code for the moment. They will be changed back once the dropdown menus no longer say "Test" and instead are filled with information from the api's.

# Checklist:
